src/palace/manager/sqlalchemy/session.py

```diff
@@ -62,18 +63,8 @@ def setup_event_listener(
         return session
 
     @classmethod
-    def sessionmaker(cls, url=None, session=None):
-        if not (url or session):
-            url = Configuration.database_url()
-        if url:
-            bind_obj = cls.engine(url)
-        elif session:
-            bind_obj = session.get_bind()
-            if not os.environ.get("TESTING"):
-                # If a factory is being created from a session in test mode,
-                # use the same Connection for all of the tests so objects can
-                # be accessed. Otherwise, bind against an Engine object.
-                bind_obj = bind_obj.engine
```

Comment:
This was moved to pytest fixtures as well, so we no longer have to rely on TESTING env var.